src/DetailsView/components/export-dialog.tsx

```diff
@@ -38,7 +40,8 @@ export const ExportDialog = NamedSFC<ExportDialogProps>('ExportDialog', props =>
         props.onDescriptionChange(value);
     };
 
-    const encodedHtml = encodeURIComponent(props.html);
+    const blob = new Blob([props.html], { type: 'text/html' });
```

Comment:
I'm wondering if this deserve to be part of the `deps` of this class, so instead of using `new` we called the function from the `deps`. I'm a little concerned about how will this work on electron for example. I'm also concern on how this affect test (not sure about performance on creating blobs). Having this as a dependency allows to mock the dependency, and thus allow us to not really create a blob on the test, just a mock of a blob (or a stub for that matter)

Reply:
If we take that approach, I wonder if it'd be better to include a `reportURLGetter(html: string)` prop that gets passed to `ExportDialog` and includes the blob creation & `windowUtils.createObjectURL`. Then the `ExportDialog` test can include a mock URL. How do you feel about this @smoralesd?

Reply:
I think that would work even better! I think the name of the property should be verb-base, something in the lines of `getReportUrl` (just turning around your original wording). It also make sense to abstract away the fact that we are creating a blob (as it could be anything we could open through the browser, right)